codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
Hi uarite:
I did not see this warning in my machine. Do I need turn on some compile flag? Could we add some compile flag to treat this as error in older gcc? I do not want some one add  a unittest in old gcc with out error, then break other one who use new gcc.